src/libraries/System.Private.CoreLib/src/System/Convert.cs

```diff
@@ -2336,7 +2339,30 @@ public static string ToBase64String(ReadOnlySpan<byte> bytes, Base64FormattingOp
             }
 
             bool insertLineBreaks = (options == Base64FormattingOptions.InsertLineBreaks);
-            string result = string.FastAllocateString(ToBase64_CalculateAndValidateOutputLength(bytes.Length, insertLineBreaks));
+            int outputLength = ToBase64_CalculateAndValidateOutputLength(bytes.Length, insertLineBreaks);
```

Comment:
@EgorBo, do we need something similar to this change for Convert.ToBase64CharArray?

Reply:
@stephentoub makes sense!